cmd/micromdm/serve.go

```diff
@@ -390,6 +395,7 @@ func serve(args []string) error {
 		r.Handle("/v1/users", apiAuthMiddleware(*flAPIKey, applyAPIHandlers.ApplyUserhandler)).Methods("PUT")
 		r.Handle("/v1/users", apiAuthMiddleware(*flAPIKey, listAPIHandlers.ListUsersHander)).Methods("GET")
 		r.Handle("/v1/config/certificate", apiAuthMiddleware(*flAPIKey, configHandlers.SavePushCertificateHandler)).Methods("PUT")
+		r.Handle("/v1/backup", listAPIHandlers.BackupHandler).Methods("GET")
```

Comment:
this needs to be authenticated, see the `apiAuthMiddleware` usage in the handlers right above.